scripts/ci/libraries/_parallel.sh

```diff
@@ -82,9 +85,9 @@ function parallel::monitor_loop() {
               continue
             fi
 
-            echo "${COLOR_BLUE}### The last lines for ${parallel_process} process: ${directory}/stdout ###${COLOR_RESET}"
+            echo "${COLOR_BLUE}### The last ${PARALLEL_TAIL_LENGTH} lines for ${parallel_process} process: ${directory}/stdout ###${COLOR_RESET}"
             echo
-            tail -2 "${directory}/stdout" || true
+            tail "-${PARALLEL_TAIL_LENGTH}" "${directory}/stdout" || true
```

Comment:
`tail -s` is sleep. Didn't you mean `tail -n`?

Reply:
No -s there :)

Reply:
.... I _swore_ I saw that as `-s${...`

👓

Reply:
`tail -<NUMBER>` works same as `tail -n <NUMBER>`

Reply:
Yeah, I use that all the time. I just can't read.